src/main/java/mate/service/DriverService.java

```diff
@@ -0,0 +1,7 @@
+package mate.service;
+
+import mate.model.Driver;
+
+public interface DriverService extends GenericService<Driver> {
+    Driver getByLogin(String login);
```

Comment:
you method mast be Optional<Driver> findByLogin(String login). it is from the condition of the task